onnxruntime/test/onnx/TestCase.h

```diff
@@ -19,6 +19,7 @@ class ValueInfoProto;
 //One test case can contain multiple test data(input/output pairs)
 class ITestCase {
  public:
+  ITestCase() : dropout_test(false){};
```

Comment:
This is a test case interface... ITestCase... 
Introducing such flag can't be a good design pattern